install.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 # Author: Alexander Epstein https://github.com/alexanderepstein
 currentVersion="1.23.0"
-declare -a tools=(bak2dvd bash-snippets cheat cloudup crypt cryptocurrency currency geo lyrics meme movies newton pwned qrify short siteciphers stocks taste todo transfer weather ytview)
+declare -a tools=(bak2dvd bash-snippets cheat cloudup crypt cryptocurrency currency geo lyrics meme movies newton pwned gist qrify short siteciphers stocks taste todo transfer weather ytview)
```

Comment:
Can we also add this to the uninstall script as well?

Reply:
fixed in 3e794b4